charts/shoot-core/components/charts/vpn-shoot/templates/deployment.yaml

```
@@ -1,117 +1,9 @@
-    type: RollingUpdate
-  selector:
-    matchLabels:
-      app: vpn-shoot
-  template:
-    metadata:
-      annotations:
-        scheduler.alpha.kubernetes.io/critical-pod: ''
-{{ include "vpn-shoot.deployment.annotations" . | indent 8 }}
-{{- if .Values.podAnnotations }}
-{{ toYaml .Values.podAnnotations | indent 8 }}
-{{- end }}
-      labels:
-        origin: gardener
-        gardener.cloud/role: system-component
-        app: vpn-shoot
-        type: tunnel
-    spec:
-      automountServiceAccountToken: false
-      serviceAccountName: vpn-shoot
-      priorityClassName: system-cluster-critical
-      dnsPolicy: Default # make sure to not use the coredns for DNS resolution.
-      nodeSelector:
-        worker.gardener.cloud/system-components: "true"
-      tolerations:
-      - key: CriticalAddonsOnly
-        operator: Exists
-      containers:
-      - name: vpn-shoot
-      {{- if .Values.reversedVPN.enabled }}
-        image: {{ index .Values.images "vpn-shoot-client" }}
-      {{- else }}
-        image: {{ index .Values.images "vpn-shoot" }}
-      {{- end }}
-        imagePullPolicy: IfNotPresent
-        env:
-      {{- if .Values.reversedVPN.enabled }}
-        - name: ENDPOINT
-          value: {{ .Values.endpoint }}
-        - name: OPENVPN_PORT
-          value: {{ .Values.port | quote }}
-        - name: REVERSED_VPN_HEADER
-          value: {{ .Values.reversedVPN.header }}
-      {{- end }}
-        - name: SERVICE_NETWORK
-          value: {{ .Values.serviceNetwork }}
-        - name: POD_NETWORK
-          value: {{ .Values.podNetwork }}
-        {{- if .Values.nodeNetwork }}
-        - name: NODE_NETWORK
-          value: {{ .Values.nodeNetwork }}
-        {{- end }}
-        securityContext:
-          privileged: true
-          capabilities:
-            add:
-            - NET_ADMIN
-        resources:
-          requests:
-            cpu: 100m
-            memory: 100Mi
-          limits:
-          {{- if .Values.global.vpaEnabled }}
-            cpu: 400m
-            memory: 400Mi
-          {{- else }}
-            cpu: "1"
-            memory: 1Gi
-          {{- end }}
-        volumeMounts:
-        - mountPath: /srv/secrets/vpn-shoot
-          name: vpn-shoot
-        - mountPath: /srv/secrets/tlsauth
-          name: vpn-shoot-tlsauth
-      {{- if not .Values.reversedVPN.enabled }}
-        - mountPath: /srv/secrets/dh
-          name: vpn-shoot-dh
-      {{- end }}
-      volumes:
-      - name: vpn-shoot
-        secret:
-          secretName: {{ include "vpn-shoot.certs.name" . }}
-          defaultMode: 0400
-      - name: vpn-shoot-tlsauth
-        secret:
-          secretName: {{ include "vpn-shoot.tls-auth.name" . }}
-          defaultMode: 0400
-      {{- if not .Values.reversedVPN.enabled }}
-      - name: vpn-shoot-dh
-        secret:
-          secretName: {{ include "vpn-shoot.dh.name" . }}
-          defaultMode: 0400
-      {{- end }}
+    resources.gardener.cloud/mode: Ignore
```

Can you leave 1 empty line at the end of the file?

```suggestion
    resources.gardener.cloud/mode: Ignore

```